src/store/modules/Logs/DumpsStore.js

```diff
@@ -110,22 +112,24 @@ const DumpsStore = {
         })
         .catch((error) => {
           const errorMsg = error;
+
           if (
-            error.response?.data?.error?.code ===
-            'Base.1.13.0.ResourceInStandby'
+            REGEX_MAPPINGS.ResourceInStandby.test(
+              error.response?.data?.error?.code
+            )
           ) {
             throw new Error(i18n.t('pageDumps.toast.errorPhypInStandby'));
           }
-          switch (errorMsg) {
-            case 'Base.1.8.1.ActionParameterUnknown':
+          switch (true) {
```

Comment:
Why this Switch is true?

Reply:
I understand that adding true will allow us to condition checks in case unlike using string. This change makes sense.